extensions/amp-skimlinks/0.1/test/test-skim-options.js

```diff
@@ -0,0 +1,57 @@
+import {getAmpSkimlinksOptions} from '../skim-options';
+import helpersFactory from './helpers';
+
+describes.fakeWin(
+    'Skim Options',
+    {
+      amp: {
+        extensions: ['amp-skimlinks'],
+      },
+    },
+    env => {
+      let helpers;
+      let docInfo;
+      beforeEach(() => {
+        helpers = helpersFactory(env);
+      });
+
+      beforeEach(() => {
+        docInfo = {canonicalUrl: 'https://mydomain.com/test'};
+      });
+
+      afterEach(() => {
+        env.sandbox.restore();
+      });
+
+      describe('excluded-domains', () => {
+        it('Should exclude domains specified in the option', () => {
+          const element = helpers.createAmpSkimlinksElement({
+            'publisher-code': '123X123',
+            'excluded-domains': ' www.merchant1.com   merchant2.com  ',
+          });
+          const options = getAmpSkimlinksOptions(element, docInfo);
+
+          expect(options.excludedDomains).to.include
+              .members(['merchant1.com', 'merchant2.com']);
```

Comment:
Perhaps add one or two from `GLOBAL_DOMAIN_BLACKLIST` and `internalDomains` to make sure they're concatenated